packages/synapse-react-client/test/containers/widgets/Checkbox.test.tsx

```diff
@@ -34,20 +37,11 @@ describe('basic function', () => {
     expect(screen.getByRole<HTMLInputElement>('checkbox').checked).toBe(false)
   })
 
-  it('should call callbackFn on change with correct params and change the value', async () => {
-    render(<Checkbox {...props} checked={false} />)
-
-    await userEvent.click(screen.getByRole('checkbox'))
-    expect(mockCallback).toHaveBeenCalledWith(true)
-
-    expect(screen.getByRole<HTMLInputElement>('checkbox').checked).toBe(true)
-  })
```

Comment:
This basically tested that the Checkbox had its own internal state. This is no longer true, so test removed!